src/graphql/__tests__/util.js

```diff
@@ -137,7 +137,6 @@ if (process.env.GCS_BUCKET_NAME) {
           "status": "SUCCESS",
           "text": "排
         汗
-        汗和排尿的差別
```

Comment:
Note: this paragraph only has confidence of 0.78, thus filtered out from the end result.
![image](https://github.com/cofacts/rumors-api/assets/108608/9610a41c-1402-4031-94e0-d8ad9306e995)